source/reference/verification/custom-manual.txt

```diff
@@ -0,0 +1,86 @@
+.. _c2c-custom-manual-verification:
+
+==================================
+Custom or Manual Verification Page
+==================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. _c2c-document-comparison: 
+
+Document Comparison
+-------------------
+
+You can verify sync by comparing documents on the source and destination 
+clusters. 
+
+Write a script that queries collections on the source cluster and then checks 
+that the correct documents, indexes, collections, metadata, and views exist with
+the same values on the destination cluster.
+
+Before you can verify data transfer with this method, ``mongosync`` must be in 
+the ``COMMITTED`` state.
+
+.. _c2c-index-comparison: 
+
+Index Comparison
+----------------
+
+To verify the transfer of indexes, run the :method:`db.collection.getIndexes` 
+method on the source and destination clusters and compare the results. 
```

Comment:
```suggestion
To verify the transfer of indexes, run :method:`db.collection.getIndexes`
on the source and destination clusters and compare the results. 
```
In general, you can usually shorten "click the X button," "run the X command," to just "click X" or "run X."